extension/persistence/eclipselink/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -98,26 +99,42 @@ public PolarisEclipseLinkMetaStoreSessionImpl(
       @Nullable String confFile,
       @Nullable String persistenceUnitName,
       @Nonnull PrincipalSecretsGenerator secretsGenerator) {
+    this(
+        createEntityManagerFactory(realmContext, confFile, persistenceUnitName),
+        store,
+        storageIntegrationProvider,
+        secretsGenerator);
     LOGGER.debug(
         "Creating EclipseLink Meta Store Session for realm {}", realmContext.getRealmIdentifier());
```

Comment:
Minor: I think this comment should be placed before `createEntityManagerFactory()`

Reply:
Java does not allow plain code statements before `this()` or `super()`.

@flyrain : Are you ok to merge like this?

Reply:
should we just remove the log?

Reply:
ok, will do presently

Reply:
removed